cmd/layotto/main.go

```diff
@@ -441,6 +442,9 @@ func NewRuntimeGrpcServer(data json.RawMessage, opts ...grpc.ServerOption) (mgrp
 			runtime_sequencer.NewFactory("mysql", func() sequencer.Store {
 				return sequencer_mysql.NewMySQLSequencer(log.DefaultLogger)
 			}),
+			runtime_sequencer.NewFactory("snowflake", func() sequencer.Store {
```

Comment:
Please modify `cmd/layotto_multiple_api/main.go` and `cmd/layotto_without_xds/main.go`

Reply:
got it